### PR TITLE
Regenerate code and run tests for Olivier's PR 880

### PR DESCRIPTION
**Pull request by valassi**
This is a WIP PR to test Olivier's #880 and complete it with regeneration of all processes

It shows that #856 is indeed fixed.

Keep as WIP while Olivier completes #880 and until that is merged.

### CODE REVIEW

**Comment by valassi** [comment]
The tests of the old #880 are good

image

I will now update to OLivier's latest 880, force push and rerun

**Comment by valassi** [comment]
Ok the CI has passed. (But I will disable the bypasses so that it fails also on known issues 

This is good to go for me. Can be merged after merging #880. @oliviermattelaer can you have a look? (It only adds tests and code generation, the rest is identical to your #880). Thanks!

image

File: epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/model_handling.py
Comment (oliviermattelaer) [inline epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/model_handling.py:1526]:
```suggestion
```
looks like I forget to remove one debug statement. So feel free to remove it (or we will clean those anyway one day in the future :-)

**Review by oliviermattelaer — APPROVED** [review]
Perfect,

Thanks,

Olivier

File: epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/model_handling.py
Comment (valassi) [reply epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/model_handling.py]: Very good thanks, change committed

**Comment by valassi** [comment]
Thanks Olivier! Merging now

**Pull request merged**
